llmproxy/models/llama2.py

```diff
@@ -0,0 +1,63 @@
+import requests
+from llmproxy.models.base import BaseChatbot, CompletionResponse
+from llmproxy.utils.enums import BaseEnum
+
+class Llama2Model(str, BaseEnum):
+    LLAMA_2_7B="Llama-2-7b-chat-hf"
+    LLAMA_2_13B="Llama-2-13b-chat-hf"
+    LLAMA_2_70B="Llama-2-70b-chat-hf"
+
+
+
+class Llama2(BaseChatbot):
+
+    def __init__(
+        self,
+        prompt: str = "",
+        system_prompt: str = "Answer politely",
+        api_key: str = "",
+        temperature: float = 1.0,
+        model: Llama2Model = Llama2Model.LLAMA_2_7B
+        ) -> None:
+        self.system_prompt = system_prompt
+        self.prompt = prompt
+        self.api_key = api_key
+        self.temperature = temperature
+        self.model = model
+
+    def get_completion(self) -> CompletionResponse:
+        if self.prompt == "":
+            return self._handle_error(
+                exception="No prompt detected", error_type="InputError"
+            )
+        if self.model not in Llama2Model:
+            return self._handle_error(
+                exception="Invalide Model", error_type="InputError"
```

Comment:
- Typo
- The error should also list out all of the models that are available (reference main branch openai file or victor's branch for mistral)